topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md

```diff
@@ -0,0 +1,842 @@
+
+## Get data
+
+The first step of our analysis consists of retrieving the RNA-seq datasets from Zenodo and organizing them into collections.
+
+> <hands-on-title>Retrieve miRNA-Seq and mRNA-Seq datasets</hands-on-title>
+>
+> 1. Create a new history for this tutorial
+> 2. Import the files from Zenodo:
+>
+>    - Open the file {% icon galaxy-upload %} __upload__ menu
+>    - Click on __Rule-based__ tab
+>    - *"Upload data as"*: `Collection(s)`
+>    - Copy the following tabular data, paste it into the textbox and press <kbd>Build</kbd>
+>
+>      ```
+>      SRR9050437	Health tissue	{{ page.zenodo_link }}/files/SRR9050437_forward.fastqsanger.gz	fastqsanger.gz	1
+>      SRR9050437	Health tissue	{{ page.zenodo_link }}/files/SRR9050437_reverse.fastqsanger.gz	fastqsanger.gz	2
+>      SRR9050438	Health tissue	{{ page.zenodo_link }}/files/SRR9050438_forward.fastqsanger.gz	fastqsanger.gz	1
+>      SRR9050438	Health tissue	{{ page.zenodo_link }}/files/SRR9050438_reverse.fastqsanger.gz	fastqsanger.gz	2
+>      SRR9050439	Health tissue	{{ page.zenodo_link }}/files/SRR9050439_forward.fastqsanger.gz	fastqsanger.gz	1
+>      SRR9050439	Health tissue	{{ page.zenodo_link }}/files/SRR9050439_reverse.fastqsanger.gz	fastqsanger.gz	2
+>      SRR9050440	Cancer tissue	{{ page.zenodo_link }}/files/SRR9050440_forward.fastqsanger.gz	fastqsanger.gz	1
+>      SRR9050440	Cancer tissue	{{ page.zenodo_link }}/files/SRR9050440_reverse.fastqsanger.gz	fastqsanger.gz	2
+>      SRR9050441	Cancer tissue	{{ page.zenodo_link }}/files/SRR9050441_forward.fastqsanger.gz	fastqsanger.gz	1
+>      SRR9050441	Cancer tissue	{{ page.zenodo_link }}/files/SRR9050441_reverse.fastqsanger.gz	fastqsanger.gz	2
+>      SRR9050442	Cancer tissue	{{ page.zenodo_link }}/files/SRR9050442_forward.fastqsanger.gz	fastqsanger.gz	1
+>      SRR9050442	Cancer tissue	{{ page.zenodo_link }}/files/SRR9050442_reverse.fastqsanger.gz	fastqsanger.gz	2
+>      ```
+>
+>    - From **Rules** menu select `Add / Modify Column Definitions`
+>       - Click `Add Definition` button and select `List Identifier(s)`: column `A`
+>
+>         > <tip-title>Can't find <i>List Identifier</i>?</tip-title>
+>         > Then you've chosen to upload as a 'dataset' and not a 'collection'. Close the upload menu, and restart the process, making sure you check *Upload data as*: **Collection(s)**
+>         {: .tip}
+>
+>       - Click `Add Definition` button and select `Collection Name`: column `B`
+>       - Click `Add Definition` button and select `URL`: column `C`
+>       - Click `Add Definition` button and select `Type`: column `D`
+>       - Click `Add Definition` button and select `Pair-end Indicator`: column `E`
+>
+>    - Click `Apply` and press <kbd>Upload</kbd>
+>
+> {% snippet faqs/galaxy/datasets_add_tag.md type="name" %}
```

Comment:
This is unclear do you want users to add tags?

Reply:
Yes, but since now both collections have been merged, it is not necessary here at all.